BitcoinClicker/game.js

```
@@ -0,0 +1,1132 @@
+                this.gameState.marketTrend = 'down';
+                break;
+                
+            case 'marketBoom':
+                this.gameState.marketPrice *= 1.3;
+                Utils.createNotification('Market Boom!', 'Bitcoin price surged 30%!', 'success');
+                this.gameState.marketTrend = 'up';
+                break;
+                
+            case 'halvening':
+                // Temporary boost to click power
+                const bonusHashes = this.getClickPower() * 100;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)